src/distributions/official_builds/official_builds.ts

```diff
@@ -100,9 +100,9 @@ export default class OfficialBuilds extends BaseDistribution {
             `Received HTTP status code ${err.httpStatusCode}. This usually indicates the rate limit has been exceeded`
           );
         } else {
-          core.info(err.message);
+          core.info((err as Error).message);
```

Comment:
If the compiler does not detect the 'Error' class, shouldn't we dynamically check it, either with a type guard or as simple as (err.message && message || something)?

Reply:
I think it is unnecessary because if it was not an error initially, the code related to `core.info` will fail because it accepts only strings. It is unexpected behaviour that string is thrown instead of Error.